### PR TITLE
Edits anothertest

### PR DESCRIPTION
**Pull request by ThadHouse**

### CODE REVIEW

File: AnotherTest [hunk 1]
```diff
@@ -12,6 +12,10 @@
 #include <memory>
 #include <vector>
 
+
```
Comment (CoolBots) [inline AnotherTest:15]: 1 too many new lines

File: AnotherTest [hunk 2]
```diff
@@ -12,6 +12,10 @@
 #include <memory>
 #include <vector>
 
+
+
+
```
Comment (CoolBots) [inline AnotherTest]: Yeah still too many

**Review by CoolBots — CHANGES_REQUESTED** [review]
There needs to be things fixed

File: AnotherTest (hunk 2 above)
Comment (ThadHouse) [reply AnotherTest]: Fixed

File: AnotherTest (hunk 1 above)
Comment (ThadHouse) [reply AnotherTest:15]: Fixed

**Review by CoolBots — APPROVED** [review]

**Pull request merged**